.idea/modules.xml

```diff
@@ -6,7 +6,6 @@
       <module fileurl="file://$PROJECT_DIR$/.idea/Clean_Marvel_Kotlin-master.iml" filepath="$PROJECT_DIR$/.idea/Clean_Marvel_Kotlin-master.iml" />
       <module fileurl="file://$PROJECT_DIR$/.idea/Clean_Marvel_Kotlin_globant.iml" filepath="$PROJECT_DIR$/.idea/Clean_Marvel_Kotlin_globant.iml" />
       <module fileurl="file://$PROJECT_DIR$/app/app.iml" filepath="$PROJECT_DIR$/app/app.iml" />
-      <module fileurl="file://$PROJECT_DIR$/app/app.iml" filepath="$PROJECT_DIR$/app/app.iml" />
     </modules>
   </component>
 </project>
```

Comment:
to avoid this annoying  git error, configure android studio like this
https://stackoverflow.com/questions/24093609/how-can-i-configure-android-studio-to-add-new-line-at-end-of-file

Reply:
Done! Thanks